p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/Steps/EnterPhoneNumber/ChoosePhoneCode/SelectableCountry.swift

```diff
@@ -8,16 +8,37 @@ struct SelectableCountry: Hashable {
 }
 
 extension Array where Element == SelectableCountry {
-    func filteredByKeyword(keyword: String) -> Self {
+    func filteredAndSorted(byKeyword keyword: String = "") -> Self {
         var countries = self
+        if keyword.isEmpty {
+            return countries.sorted(by: { $0.value.name < $1.value.name })
+        }
         if !keyword.isEmpty {
-            let searchText = keyword.lowercased()
-            countries = countries.filter { country in
-                country.value.name.lowercased().contains(searchText) ||
-                    country.value.code.lowercased().contains(searchText) ||
-                    searchText.contains(country.value.emoji ?? "") ||
-                    country.value.dialCode.contains(searchText)
-            }
+            let keyword = keyword.lowercased()
+            countries = countries
+                .filter { country in
+                    country.value.name.lowercased().contains(keyword) ||
+                        country.value.code.lowercased().contains(keyword) ||
+                        keyword.contains(country.value.emoji ?? "") ||
+                        country.value.dialCode.contains(keyword)
+                }
+                .sorted(by: { c1, c2 in
+                    let name1 = c1.value.name.lowercased()
+                    let name2 = c2.value.name.lowercased()
+
+                    // Sort:
+                    // First by name that starts with keyword, then by alphabet
```

Comment:
if sort includes keyword logic then we don't need to remove it from the list and insert again higher? ChoosePhoneCodeViewController, line 88